test/specs/sObjectsDefinitions.e2e.ts

```diff
@@ -203,11 +214,34 @@ describe('SObjects Definitions', async () => {
     const treeViewSection = await content.getSection(projectName);
     await expect(treeViewSection).not.toEqual(undefined);
 
-    // Verify if 'standardObjects' folder is in side panel
-    const standardObjectsTreeItem = (await treeViewSection.findItem(
-      'standardObjects'
-    )) as DefaultTreeItem;
+    // Verify if '.sfdx' folder is in side panel
+    const sfdxTreeItem = (await treeViewSection.findItem('.sfdx')) as DefaultTreeItem;
+    await expect(sfdxTreeItem).not.toEqual(undefined);
+    await sfdxTreeItem.expand();
+    await expect(await sfdxTreeItem.isExpanded()).toBe(true);
+    await utilities.pause(utilities.Duration.seconds(1));
+
+    // Verify if 'tools' folder is within '.sfdx'
+    const toolsTreeItem = (await sfdxTreeItem.findChildItem('tools')) as TreeItem;
+    await expect(toolsTreeItem).not.toEqual(undefined);
+    await toolsTreeItem.expand();
+    await expect(await toolsTreeItem.isExpanded()).toBe(true);
+    await utilities.pause(utilities.Duration.seconds(1));
+
+    // Verify if 'sobjects' folder is within 'tools'
+    let sobjectsTreeItem = await utilities.getVisibleChild(sfdxTreeItem, 'sobjects');
+    await expect(sobjectsTreeItem).not.toEqual(undefined);
+    sobjectsTreeItem = sobjectsTreeItem!;
+
+    await sobjectsTreeItem.expand();
+    await expect(await sobjectsTreeItem.isExpanded()).toBe(true);
+    await utilities.pause(utilities.Duration.seconds(1));
+
+    // Verify if 'standardObjects' folder is within 'sobjects'
+    let standardObjectsTreeItem = await utilities.getVisibleChild(sfdxTreeItem, 'standardObjects');
     await expect(standardObjectsTreeItem).not.toEqual(undefined);
+    standardObjectsTreeItem = standardObjectsTreeItem!;
```

Comment:
I understand that the expect does not provide TS with enough informaion to continue without applying the `!`, but use of `!` is discouraged. 
```suggestion
    if (!standardObjectsTreeItem) {
      fail('Could not locate standardObjects in tree view');
    }
    ...
```